it/src/main/java/org/corfudb/universe/util/IpTablesUtil.java

```diff
@@ -21,8 +21,28 @@ public static String dropInput(String ipAddress) {
      * @param ipAddress ip address to drop packages
      * @return command line
      */
-    public static String dropOutput(String ipAddress) {
-        return "iptables -A OUTPUT -d " + ipAddress + " -j DROP";
+    public static String[] dropOutput(String ipAddress) {
+        return new String[] {"iptables", "-A", "OUTPUT", "-d", ipAddress, "-j", "DROP"};
+    }
+
+    /**
+     * Recover the drop input rule
+     *
+     * @param ipAddress ip address to recover the drop input rule
+     * @return command line
+     */
+    public static String[] recoverDropInput(String ipAddress) {
```

Comment:
recover is a little confusing. We are deleting a rule added before. maybe deleteDropInput or revertDropInputRule?

Reply:
Done.